src/auditlog/middleware.py

```diff
@@ -5,7 +5,10 @@
 
 from django.conf import settings
 from django.db.models.signals import pre_save
-from django.utils.functional import curry
+try:
+    from functools import partial
+except ImportError:
+    from django.utils.functional import curry as partialmethod
```

Comment:
`partialmethod` is not used in the code, should it be `partial` instead ?

Reply:
Thanks for the check on this @pcamelo. I see that this [fork made the change](https://github.com/vinaygode/django-auditlog/commit/cc87b00d60866f986bd21e993dfab18fbf1ab89d) from @vinaygode. 

I haven't tried it with Django 2.2, but that seems like right way to handle curry and partial. This is also recommended on [Django 3 release notes](https://docs.djangoproject.com/en/3.0/releases/3.0/).